### DISCUSSION TITLE
image with caption for alignment

### DISCUSSION THREAD

**Discussion by mdeliw** [squidfunk/mkdocs-material#3815] [Q&A] [answered]
As mentioned [here](https://squidfunk.github.io/mkdocs-material/reference/images/?h=image#image-alignment), the following works:

```
<figure markdown>
  ![Image title](https://dummyimage.com/600x400/){ width="300" }
  <figcaption>Image caption</figcaption>
</figure>
```

However, the above format doesn't work for images within the repository, or I am obviously doing something wrong. How can this work?

```
<figure markdown>
    ![Image title](../images/odoo/odoo-header-area.jpg){ width="512" }
    <figcaption>Image caption</figcaption>
</figure>
```

mkdocs.yml has the extensions:

```
markdown_extensions:
  - abbr
  - admonition
  - attr_list
  - md_in_html
```

**Comment by mdeliw** [comment, marked as answer]
This actually works. Please ignore my ask.

**Reply by squidfunk** [reply]
For future users, could you please explain what fixed your issue?

**Reply by mopa** [reply]
Yes please, I have the same issue.

**Reply by janosmurai** [reply]
The problem is with the number of whitespaces used for indentation. 
The example
```markdown
<figure markdown>
  ![Image title](https://dummyimage.com/600x400/){ width="300" }
  <figcaption>Image caption</figcaption>
</figure>
```
gets generated to 
```html
<figure>
  <p><img alt="Image title" src="https://dummyimage.com/600x400/" width="300">
    </p>
  <figcaption>Image caption</figcaption>
</figure>
```
But for example if 4 spaces are used like
```markdown
<figure markdown>
    ![Image title](https://dummyimage.com/600x400/){ width="300" }
    <figcaption>Image caption</figcaption>
</figure>
```
the result is
```html
<figure>
<pre><code>![Image title](https://dummyimage.com/600x400/){ width="300" }
&lt;figcaption&gt;Image caption&lt;/figcaption&gt;
</code></pre>
</figure>
```

**Comment by rwoldu** [comment]
Just a small remark:
Using the [lightbox plugin](https://squidfunk.github.io/mkdocs-material/reference/images/#lightbox), the [originally suggested formatting for captions](https://squidfunk.github.io/mkdocs-material/reference/images/#image-captions) works

But using lightbox, the alignment of the images changes to center - is this intended?
Can I have captions without the centered alignment?

@mdeliw may I ask what was meant by "alignment" in your your title?
